src/Security/MemberAuthenticator/Authenticator.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace SilverStripe\Security\MemberAuthenticator;
+
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\Session;
+use SilverStripe\Dev\Debug;
+use SilverStripe\ORM\ValidationResult;
+use InvalidArgumentException;
+use SilverStripe\Security\Authenticator as BaseAuthenticator;
+use SilverStripe\Security\Security;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\LoginAttempt;
+
+/**
+ * Authenticator for the default "member" method
+ *
+ * @author Sam Minnee <sam@silverstripe.com>
+ * @author Simon Erkelens <simonerkelens@silverstripe.com>
+ */
+class Authenticator implements BaseAuthenticator
```

Comment:
This class differs from SilverStripe\Security\Authenticator only by namespace; It's a little vague; Can you please rename this? Maybe just call it MemberAuthenticator, or LoginAuthenticator? You can pick the name. :)